tests/gutenberg_ro.e2e.spec.js

```diff
@@ -0,0 +1,360 @@
+/**
+ * legacy-ray_charles.e2e.spec.js : End-to-end tests implemented with Selenium WebDriver and Mocha
+ *
+ * Copyright 2023 Jaifroid and contributors
+ * Licence GPL v3:
+ *
+ * This file is part of Kiwix.
+ *
+ * Kiwix is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Kiwix is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Kiwix (file LICENSE-GPLv3.txt).  If not, see <http://www.gnu.org/licenses/>
+ */
+
+// eslint-disable-next-line no-unused-vars
+import { By, Key, until, WebDriver } from 'selenium-webdriver';
+// import firefox from 'selenium-webdriver/firefox.js';
+import assert from 'assert';
+import paths from './paths.js';
+import fs from 'fs';
+/* eslint-disable camelcase */
+/* global describe, it */
+
+const port = process.env.BROWSERSTACK_LOCAL_IDENTIFIER ? '8099' : '8080';
+
+// Set the archives to load
+/**
+ *  Run the tests
+ * @param {WebDriver} driver Selenium WebDriver object
+ * @param {Array} modes Array of modes to run the tests in
+ * @returns {void}
```

Comment:
I previously suggested you change this to something like:

`@returns {Promise} A Promise that resolves when the test is complete`